src/panels/config/script/ha-script-editor.js

```diff
@@ -281,7 +286,9 @@ class HaScriptEditor extends LocalizeMixin(NavigateMixin(PolymerElement)) {
   }
 
   async _delete() {
-    if (!confirm("Are you sure you want to delete this script?")) {
+    if (
+      !confirm("[[localize('ui.panel.config.script.editor.delete_confirm')]]")
```

Comment:
Remove the "", it is no longer a string
```suggestion
      !confirm([[localize('ui.panel.config.script.editor.delete_confirm')]])
```

Reply:
And use `this.hass.localize`, as it is not used inside the Polymer render.